assets/activation-scripts/activate.d/flox-env-helper.awk

```diff
@@ -0,0 +1,199 @@
+#!@nawk@/bin/nawk -f
```

Comment:
This script would be a good candidate for a `flox-activations` subcommand.

Reply:
Big +1

Reply:
We get antsy when we want to add a new language, and recently we've added Perl for the environment builder, Awk for this script, Make (shudder) for the package builder, and C for `t3`. I also already have this logic in the Rust experiment I did.